### ISSUE TITLE
Add forgot password option at login page

### ISSUE THREAD

**Issue by sarthaxtic** [VesperAkshay/qr-code-generator#130]
**Problem Statement**
No forgot password option present at login page

**Proposed Solution**
Add forgot password option at login page

**Alternatives Considered**
Describe any alternative solutions or features you've considered.

**Additional context**

**Comment by aryansoni13** [comment]
I  would like to work on this, kindly assign me this task

**Comment by VesperAkshay** [comment]
Its already been done

**Comment by VanshKirtishahi** [comment]
Hello!

Is there an existing issue for this?
I have searched the existing issues and found issue #197 regarding the confirmation message on login.

Feature Description:
I would like to implement a confirmation message that appears after a user successfully logs in to the application.

Use Case:
This feature will inform users that they have successfully logged in, enhancing the user experience.

Benefits:

Provides immediate feedback to the user.
Helps prevent confusion and improves usability.
Add Screenshots:
No screenshots are available at this time, but I can provide mockups if needed.

Priority:
I consider this a high priority feature, as it directly affects user experience.

Record:
I have read the Contributing Guidelines.
I am a GSSOC'24 contributor.
I am a Hactoberfest contributor
I have starred the repository.

Thank you for considering my request! I look forward to your feedback.
